<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>data_Process</class>
 <widget class="QDialog" name="data_Process">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>数据处理</string>
  </property>
  <property name="windowIcon">
   <iconset resource="image.qrc">
    <normaloff>:/image/image/001.jpg</normaloff>:/image/image/001.jpg</iconset>
  </property>
  <widget class="QPushButton" name="select_files">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>340</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>批量选择文件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="start_process">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>340</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>开始处理</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>1031</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>180</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>180</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="select_algorithm">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>340</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>处理配置</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>339</y>
     <width>691</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="p_chart" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>651</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
